certs/service_test.go

```diff
@@ -1,414 +0,0 @@
-// Copyright (c) Mainflux
```

Comment:
Do not remove tests. It's OK if they fail for the time being.

Reply:
It isn't about failing or passing. We have different testing methods. For clients, we don't use maps